File: Daily Challanges/2026-03-March/2026-03-18.js
function largestNumber(s) {
  const separators = ",;:!?";
  const numbers = [];
  let current = "";
  for (let char of s) {
    if (separators.includes(char)) {
      numbers.push(Number(current));
      current = "";
    } else {
      current += char;
    }
  }
  numbers.push(Number(current));
  return Math.max(...numbers);
}

console.log(largestNumber("1,2"));

File: Daily Challanges/2026-03-March/2026-03-18.py
def largest_number(s):
    seperators = ",;:!?"
    numbers = []
    current = ""
    for char in s:
        if char in seperators:
            numbers.append(float(current))
            current = ""
        else:
            current += char
    numbers.append(float(current))
    return max(numbers)

print(largest_number("1,2"))
print(largest_number("12;-50,99.9,49.1!-10.1?88?16") )